var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -50,7 +50,8 @@ class Scr(CMakePackage):
 
     variant('libyogrt', default=True,
             description="Build SCR with libyogrt for get_time_remaining.")
-    depends_on('libyogrt', when="+libyogrt")
```

Comment:
You might want to keep this line. I'm assuming `+libyogrt resource_manager=NONE` still needs a dependency on `libyogrt`

Reply:
Looks like I get the default libyogrt option `scheduler=system` when I put that back, which looks good. Committed.